contract/README.md

```diff
@@ -10,6 +10,9 @@ npx hardhat compile
 npx hardhat clean
 npx hardhat test
 npx hardhat node
-node scripts/sample-script.js
+node scripts/test/sample-test.js
```

Comment:
It's good for people to know these exist! They aren't very well-documented though - I used them while I was developing this 🤔